server/server.go

```diff
@@ -630,6 +628,13 @@ func (s *Server) ShowProcessList() map[uint64]*util.ProcessInfo {
 			rs[pi.ID] = pi
 		}
 	}
+	if s.dom != nil {
+		for connID, p := range s.dom.GetSysProcesses() {
+			pi := p.(session.Session).ShowProcess()
+			pi.ID = connID
```

Comment:
1. Do we need to check whether `pi == nil`?
2. We set `pi.ID = connID` here. Will `pi.ID` be used somewhere?

Reply:
refactored